samples/bluetooth/fast_pair/locator_tag/README.rst

```diff
@@ -134,9 +134,6 @@ User interface
 
 The user interface of the sample depends on the hardware platform you are using.
 
-Development kits
-================
-
 .. tabs::
 
    .. group-tab:: nRF52 and nRF53 DKs
```

Comment:
`.. group-tab:: Development kits` as it's done later in the doc

Reply:
It should be also aligned in other places in this doc. And also in https://github.com/nrfconnect/sdk-nrf/blob/main/samples/bluetooth/fast_pair/input_device/README.rst doc. It can be done in follow-up task. We must have missed it when adding nrf54 support.

Reply:
This was intentional because nRF54 DKs have LED 0 and Button 0 while the other nRF5 Series devices start with LED 1.

Reply:
Oh, I forgot that we have different Leds and buttons numbering. Disregard my comment.